specification/columns/listcost.md

```diff
@@ -1,10 +1,10 @@
 # List Cost
 
-List Cost represents the cost calculated by multiplying [List Unit Price](#listunitprice) and the corresponding [Pricing Quantity](#pricingquantity). List Cost is denominated in the [Billing Currency](#billingcurrency) and is commonly used for calculating savings based on various rate optimization activities, by comparing it with [Effective Cost](#effectivecost).
+List Cost represents the cost calculated by multiplying the [*list unit price*](#glossary:list-unit-price) and the corresponding [Pricing Quantity](#pricingquantity). List Cost is denominated in the [Billing Currency](#billingcurrency) and is commonly used for calculating savings based on various rate optimization activities, by comparing it with [Billed Cost](#billedcost) and [Effective Cost](#effectivecost).
 
-The ListCost column MUST be present in the billing data and MUST NOT be null. This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat), and be denominated in the BillingCurrency. When a ListUnitPrice is not null, multiplying the ListUnitPrice by PricingQuantity MUST produce the ListCost.
+The ListCost column MUST be present in the billing data and MUST NOT be null. This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat) requirements, and be denominated in the BillingCurrency. Multiplying the *list unit price* by PricingQuantity MUST produce the ListCost.
```

Comment:
Do we refer to the glossary instead of directly mentioning the actual columns in the normative paragraph, only in the List Cost and ListUnitPrice specs? Taking into account that it is normative, I would prefer the reference to the actual column.
Consider something like this:

```suggestion
The ListCost column MUST be present in the billing data and MUST NOT be null. This column MUST be of type Decimal, MUST conform to [Numeric Format](#numericformat) requirements, and be denominated in the BillingCurrency. When [ListUnitPrice](#listunitprice) is present and is not null, multiplying the ListUnitPrice by PricingQuantity MUST produce the ListCost.
```

Reply:
I was debating this. I'm good with the suggestion. 

(previously, I was debating if we should refer to the glossary entry to remove the complication of list unit price being conditional)